stores/sql/mysql/migrations/main/migration_00036_wallet_locked_utxos.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `wallet_locked_outputs` (
```

Comment:
Blocker to make sure these were manually verified

Reply:
verified manually, tested on both SQLite and MySQL - decided to change the identifier though 👍